README.md

```diff
@@ -79,6 +79,15 @@ We packaged critical extensions for compatibility with our EDC UI and general us
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+## sovity Community Edition Broker extension
+
+[//]: # (TODO: check if this is still the same version at releasing.)
+Our EDC Broker indexes EDCs to offer a convenient centralized search engine. It is part of this project since version 8.
+
+See [here](launchers/README.md) for a list of our sovity Community Edition Broker Docker images.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
We should remove this.

Honestly, we don't need to adjust the main description text just yet, which is what I would've suggested as an alternative, as the broker will soon be dissolved

Reply:
okay, removed.